recipes/latex-rubber/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "latex-rubber" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/latex-rubber-{{ version }}.tar.gz
+  sha256: 47bbf8d6511f6d85f98baebbebb7cb1a58e9e6b1f92ff09fa6d8c08a683ea665
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```

Since this doesn't need to build any source code, I *think* this can be noarch:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
```

---

That should work, thanks!